helmfile.d/snippets/defaults.yaml

```diff
@@ -444,8 +446,7 @@ environments:
               limits:
                 cpu: 1000m
                 memory: 512Mi
-        backup:
-          platform:
+        PlatformBackups:
```

Comment:
everywhere we start from  small letter. Can we keep that convention ?

Reply:
Hi Sander the issue still exists. Take a look at the `defaults.yaml` file.
The property name must be consistent